app/src/filters/components/FilterGroupForm.jsx

```diff
@@ -99,10 +94,18 @@ class FilterGroupForm extends Component {
       <div>
         <h3 className={ModalStyles.title}>Filter Group</h3>
         <div className={ModalStyles.optionsContainer}>
+          {this.props.help}
+        </div>
+        {this.props.warning &&
+          <div className={classnames(ModalStyles.warning, WarningStyles.warning)}>
+            ⚠️ {this.props.warning}
```

Comment:
I'd rather not have weird characters. Can we use something else instead, or no icon at all if needed?